testsuite/cluster-test/src/experiments/reconfiguration_test.rs

```diff
@@ -134,15 +130,12 @@ impl Experiment for Reconfiguration {
         let validator_name = self.affected_pod_name.as_bytes().to_vec();
         let timer = Instant::now();
         for i in 0..self.count / 2 {
-            let remove_txn = gen_submit_transaction_request(
-                TransactionPayload::Script(encode_remove_validator_and_reconfigure_script(
+            let remove_txn = diem_root_account.sign_with_transaction_builder(
+                TransactionFactory::new(ChainId::test()).remove_validator_and_reconfigure(
```

Comment:
Move the creation of the factory to the beginning on this function and then reuse it here and down below